label_studio/data_manager/api.py

```diff
@@ -224,11 +225,23 @@ def sync_paginate_queryset(self, queryset, request, view=None):
         self.total_annotations = Annotation.objects.filter(task_id__in=queryset, was_cancelled=False).count()
         return super().paginate_queryset(queryset, request, view)
 
+    def paginate_totals_queryset(self, queryset, request, view=None):
+        totals = queryset.values('id', 'total_predictions', 'total_annotations', 'cancelled_annotations').aggregate(
+            total_annotations=Sum('total_annotations') - Sum('cancelled_annotations'),
+            total_predictions=Sum('total_predictions'),
+        )
+        # if the total is None, set it to 0
+        # the default value of the aggregate function is None
+        self.total_annotations = totals['total_annotations'] or 0
+        self.total_predictions = totals['total_predictions'] or 0
```

Comment:
[Coalesce](https://docs.djangoproject.com/en/5.1/ref/models/database-functions/#coalesce) can be used here instead to defer this to the db.

Reply:
Good point, though in this case it is not a heavy operation and is only a default of this query.

Reply:
Agreed, just a nitpick